doc/lua_api.txt

```diff
@@ -2424,6 +2424,12 @@ Helper functions
 * `table.insert_all(table, other_table)`:
     * Appends all values in `other_table` to `table` - uses `#table + 1` to
       find new indices.
+* `table.shuffle(table, [from], [to], [random_func])`:
+    * Shuffles elements `from` to `to` in `table` in place
+    * `from` defaults to `1`
+    * `to` defaults to `#table`
+    * `random_func` defaults to `math.random`. This function receives two
+      integers as arguments and should return a random integer between them.
```

Comment:
I think it should specify that it's inclusive

Reply:
Done. I thought that people can infer this from how math.random, the default function, works, but it's probably inconvenient for inexperienced persons to look up the documentation.